README.md

```diff
@@ -5,6 +5,19 @@
 
 ## Usage
 
+```bash
+$ go get github.com/screwdriver-cd/launcher
```

Comment:
Assuming $GOPATH/bin is in your PATH (it is supposed to be), this section can be shortened to:

```bash
go get github.com/screwdriver-cd/launcher
launch --api-uri http://localhost:8080/v3 cba94a05f8aa063f4b8cfb62cbc355e0c5f02698
```